keyboards/dichotomy/keymaps/default/keymap.c

```diff
@@ -0,0 +1,517 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include QMK_KEYBOARD_H
+#include "report.h"
+#include "pointing_device.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum dichotomy_layers
```

Comment:
This is more of a nit pick, and you can ignore it if you want, but could this be reformatted to match our coding conventions (along with the other sections):
https://docs.qmk.fm/#/contributing?id=coding-conventions

Eg, it should be:
```c 
enum dichotomy_layers {
  ... 
};
```